src/module/item/move/document.js

```diff
@@ -45,7 +45,7 @@ class PTUMove extends PTUItem {
             postStab: 0,
             isStab: false,
         }
-        result.postStab = result.preStab + (this.actor?.types.includes(this.system.type) ? 2 : 0);
+        result.postStab = result.preStab + (this.actor?.types.includes(this.system.type) && !this.system.isStruggle ? 2 : 0);
```

Comment:
```suggestion
        result.postStab = result.preStab + (!this.system.isStruggle && this.actor?.types.includes(this.system.type) ? 2 : 0);
```
Even more nitpicky but it's more performant if struggles instantly return false here instead of *first* checking if they should be stab and *then* deciding if they're a struggle so it doesn't apply regardless.